### ISSUE TITLE
Sync fails for recurring event with error: [Bad item skipped <Invalid item!>]

### ISSUE THREAD

**Issue by orende** [jobisoft/TbSync#224]
## Your environment

TbSync version: 0.7.22
Thunderbird version: 60.2.1 (64-bit)
Provider for Exchange ActiveSync: 0.9

[ ] Yes, I have installed the latest available (beta) version from 
https://github.com/jobisoft/TbSync/releases 
and my issue is not yet fixed, I can still reproduce it.

## Expected behavior
Syncing a calendar with a recurring event should work.

## Actual behavior
Syncing fails with an error message.

## Steps to reproduce
1. Add recurring event of the type shown in the debug log
2. Sync calendar with outlook.office.com

To help resolving your issue, enable debug logging (TbSync Account Manager -> Help) and send me the debug.log via e-mail (use the title of your issue as subject of the email).

[tbsync_error_log.txt](https://github.com/jobisoft/TbSync/files/2791885/tbsync_error_log.txt)

**Comment by orende** [comment]
I just noticed that https://github.com/jobisoft/EAS-4-TbSync/wiki/About:-Provider-for-Exchange-ActiveSync says that "sending or responding to meeting invitations is not fully working". Perhaps that's what's causing the corruption?

**Comment by jobisoft** [comment]
Please switch to beta and report if error persists.
https://tbsync.jobisoft.dr

**Comment by jobisoft** [comment]
Wrong link. Sorry.
https://tbsync.jobisoft.de

**Comment by orende** [comment]
Alright, will try the beta and report back.

**Comment by jobisoft** [comment]
The beta is now released as stable.

**Comment by jobisoft** [comment]
Thanks, the feedback from the new version is much more helpful. Will investigate.

**Comment by jobisoft** [comment]
Hm, I was able to sync that event to my Office 365 server and to my outlook.com account. A crafted the request from hand so I send exactly what you send me. Except the redacted parts of course. 

![image](https://user-images.githubusercontent.com/5830621/52133877-902bd700-2642-11e9-8bf9-1a5852787ac5.png)

Is there anything special about the redacted parts (Organizer and Attendee)?

**Comment by orende** [comment]
No, both of them are simple ascii-compatible names, e.g. firstname.lastname@domain.com.

I ended up deleting the event and creating another one, which made the sync start working again. I'll close this issue now and open a new one if something similar happens in the future.

**Comment by fleskesvor** [comment]
I'm having the same issue, and no amount of deleting and re-creating events and tasks make the sync start working again. Is there any kind of useful information that I can provide to make it easier to debug this issue?

**Comment by jobisoft** [comment]
yes, a debug log. You can create one under help in the tbsync manager.

**Comment by fleskesvor** [comment]
Ok, I'm attaching a debug log. (Hopefully there isn't anything sensitive in it.)

[thunderbird-debug.log](https://github.com/jobisoft/TbSync/files/3161614/thunderbird-debug.log)

**Comment by jobisoft** [comment]
Thanks, I will have a look.

**Comment by fullbuffer** [comment]
@jobisoft I seem to be having the same or very similar issue.. though my event is not recurring. 

I have installed the latest beta, but still have the issue.  Looking at the info, the debug is very similar.  I've attached my debug log as well.

[thunderbird-tbsync-debug.log](https://github.com/jobisoft/TbSync/files/3254670/thunderbird-tbsync-debug.log)

**Comment by jobisoft** [comment]
Moving this over to the EAS provider. It happend again even in the latest release for TB68 :-(
